jbdevstudio/com.jboss.jbds.central.discovery/plugin.xml

```diff
@@ -9,6 +9,12 @@
 Supported by Red Hat.</description>
          <icon image48="images/redhat_48.png"></icon>
       </certification>
+      <certification id="com.jboss.jbds.discovery.certification.supported.notavailable" name="Supported - Not Available" url="https://devstudio.redhat.com/9.0/stable/updates/#central">
+         <description>Supported by Red Hat.
+
+This content is unavailable. Enable Early Access to see it.</description>
+         <icon image48="images/redhat_48.png"></icon>
```

Comment:
I understood the purpose behind a "not available" connector was to advertise that it was coming from Early Access. Where else would it be available? And if not available anywhere, what's the point of the teaser?